WDAC-Policy-Wizard/app/Resources/policyTemplates/Recommended_Driver_Blocklist.xml

```diff
@@ -2081,21 +4638,16 @@
   <CiSigners />
   <HvciOptions>0</HvciOptions>
   <Settings>
-    <Setting Provider="PolicyInfo" Key="Information" ValueName="Name">
-      <Value>
-        <String>Microsoft Windows Driver Policy</String>
-      </Value>
-    </Setting>
     <Setting Provider="PolicyInfo" Key="Information" ValueName="Id">
       <Value>
-        <String>10.0.25210.0</String>
+        <String>10.0.27685.0</String>
       </Value>
     </Setting>
-    <Setting Provider="PolicyInfo" Key="NoRevalidationUponRefresh" ValueName="NoRevalidationUponRefreshValue">
```

Comment:
Also looking at the diff, is this change OK, or should it be removed to match the previous?